docs/synthetic-tokens/glossary.md

```diff
@@ -11,7 +11,7 @@ Instead, it is only used to resolve disputes of liquidations and to settle synth
 ### Token sponsor
```

Comment:
Can't comment directly on this here, but if we are going to move the Glossary to outside of the synthetic assets section, this page should not be titled "Synthetic Tokens: Glossary"